go/tasks/logs/logging_utils.go

```diff
@@ -44,11 +45,13 @@ func GetLogsForContainerInPod(ctx context.Context, pod *v1.Pod, index uint32, na
 
 	logs, err := logPlugin.GetTaskLogs(
 		tasklog.Input{
-			PodName:       pod.Name,
-			Namespace:     pod.Namespace,
-			ContainerName: pod.Spec.Containers[index].Name,
-			ContainerID:   pod.Status.ContainerStatuses[index].ContainerID,
-			LogName:       nameSuffix,
+			PodName:           pod.Name,
+			Namespace:         pod.Namespace,
+			ContainerName:     pod.Spec.Containers[index].Name,
+			ContainerID:       pod.Status.ContainerStatuses[index].ContainerID,
+			LogName:           nameSuffix,
+			PodUnixStartTime:  pod.CreationTimestamp.Unix(),
+			PodUnixFinishTime: time.Now().Unix(),
```

Comment:
Is this relying on the fact that we will stop calling this API after the container has finished?
I guess that's less complex than trying to use `pod.Status.ContainerStatuses[index].LastTerminationState.Terminated.FinishedAt` (which will only be populated after the container terminates....)